cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -905,48 +905,21 @@ def test_advanced_components_munge_tabs(self):
         """
         Test that adding and removing specific advanced components adds and removes tabs.
         """
-        open_ended_tab = {"type": "open_ended", "name": "Open Ended Panel"}
```

Comment:
This change, while necessary to remove ORA1, subtly alters the test case here.

Previously, it tested the behavior when _multiple_ tabs were added, now it only tests the case when a _single_ tab is added.

Should we add a replacement tab?